lib/lndclient/LndClient.ts

```diff
@@ -177,7 +176,6 @@ class LndClient extends BaseClient {
     if (!this.isConnected()) {
       this.logger.info(`trying to verify connection to lnd with uri: ${this.uri}`);
       this.lightning = new LightningClient(this.uri, this.credentials);
-
```

Comment:
Personally I think this blank line improves readability so I'd prefer to keep it.